otcextensions/sdk/dcs/v1/quota.py

```diff
@@ -0,0 +1,109 @@
+# Licensed under the Apache License, Version 2.0 (the "License"); you may
+# not use this file except in compliance with the License. You may obtain
+# a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+# WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+# License for the specific language governing permissions and limitations
+# under the License.
+
+from openstack import exceptions
+from openstack import resource
+
+
+class Quota(resource.Resource):
+    """Distributed Cache Service Quota resource"""
+    resources_key = 'quotas.resources'
+    base_path = '/quota'
+
+    # capabilities
+    allow_list = True
+
+    #: Properties
+    #: Quota type, value is ``instance`` or ``ram``
+    type = resource.Body('type')
+    #: Quota amount of created instances and used memory
+    used = resource.Body('used', type=int)
+    #: Quota amount
+    quota = resource.Body('quota', type=int)
+    #: Resource unit, when ``type`` is set to ``ram`` ``GB`` is returned
+    unit = resource.Body('unit')
+    #: Max limit of instance or memory quota
+    max = resource.Body('max', type=int)
+    #: Min limit of instance or memory quota
+    min = resource.Body('min', type=int)
+
+    @staticmethod
+    def find_value_by_accessor(input_dict, accessor):
+        """Gets value from a dictionary using a dotted accessor"""
+        current_data = input_dict
+        for chunk in accessor.split('.'):
+            if isinstance(current_data, dict):
+                current_data = current_data.get(chunk, {})
+            else:
+                return None
+        return current_data
```

Comment:
you can move this method to sdk_utils

Reply:
I've put this code to new mixin class for quota.